yearn/utils.py

```diff
@@ -64,7 +64,7 @@ def contract_creation_block(address) -> int:
     if client in ['tg', 'erigon']:
         return _contract_creation_block_binary_search(address)
     else:
-        return _contract_creation_block_bigquery(address)
+        raise Exception("An erigon (fka turbogeth) node is needed correctly estimate contract creation block.")
```

Comment:
Merge `_contract_creation_block_binary_search` in this func, I believe it'll throw a missing trie error for a non-archive node.